source/multigrid/mg_transfer_prebuilt.cc

```diff
@@ -469,6 +600,11 @@ MGTransferPrebuilt<VectorType>::fill_and_communicate_copy_indices
     std::sort(copy_indices_level_mine[level].begin(), copy_indices_level_mine[level].end(), compare);
   for (unsigned int level=0; level<copy_indices_global_mine.size(); ++level)
     std::sort(copy_indices_global_mine[level].begin(), copy_indices_global_mine[level].end(), compare);
+
+  // set additional data for parallel::distributed::Vector
+  VectorType dummy;
```

Comment:
What is this dummy for? Is that for automatic template type deduction? I would prefer you just specify them.